index.html

```diff
@@ -15,6 +15,8 @@
     />
     <link rel="apple-touch-icon" href="/logo192.png" />
     <link rel="manifest" href="/manifest.json" />
+    <link rel="preconnect" href="https://rsms.me/" />
+    <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
     <link rel="stylesheet" href="https://use.typekit.net/zzv8rvm.css" />
```

Comment:
Can we please have a follow up PR that just includes these locally?

Reply:
Oh I can just do that now, no sweat

Reply:
<3